### DISCUSSION TITLE
Help! Games getting too much responses!

### DISCUSSION THREAD

**Discussion by IzumiSenaSora** [Heroic-Games-Launcher/HeroicGamesLauncher#1155] [General]
Help!... i installed GTA5 From Heroic Launcher
And trying to play games with (Proton 7.0) ....
when i try to walk forward in game with (Key -W) or go backward With (Key S) even if i stop pressing keyboard key the person in game still walking!... same goes for car riding....
Forward backward everything acting strange... it feels like i command games to walk 1 meter but it walk 3 to 4 meter....

I am using Fedora
I tried with (Wine 7, Proton Experiment) Same issue!... 
Can someone help me?
In Windows with epic games launcher it works fine

**Comment by flavioislima** [comment]
Try enabling virtual desktop on the wine settings.

**Reply by IzumiSenaSora** [reply]
I want to run with proton!...
I tested with wine just in case if it can solve my issue (which did not)...
